### ISSUE TITLE
Telegram login not working from mobile and working with issues from Desktop

### ISSUE THREAD

**Issue by damascene** [pennersr/django-allauth#3262]
I've been investigating an issue with Telegram login for sometime now and I've not succeeded to make it work from mobile. It works most of the time from Desktop but I may need to press the login button and refresh multiple time. 

Issue that when I press the login button from mobile I'm being taken to the Telegram authentication screen and after I enter my phone number and authenticate I'm returned to the login screen and nothing happen afterward.  On desktop pressing the login button few times allows me to login. 

I couldn't get any meaning full output from logs. 

I've set a docker compose image to recreate the issue that could be tested locally:
https://gitlab.com/uak/django_classified

You need to setup the following variables in the `.env` file:

```
BOT_TOKEN=
BOT_NAME=
TG_REDIRECT_LOGIN_URL=https://127.0.0.1/accounts/telegram/login/
```

**Comment by damascene** [comment]
I've discovered that it works from Desktop only for Firefox and not for Chrome, many users varying experience on mobile phones.

**Comment by pennersr** [comment]
Fixed via 89944bc2

**Comment by pennersr** [comment]
https://codeberg.org/allauth/django-allauth/issues/3262
